### ISSUE TITLE
Behavior beyond the model max distance

### ISSUE THREAD

**Issue by willclarkson** [willclarkson/rubinCadenceScratchWIC#18]
We should add the capability to correctly handle distances greater than the model maximum distance. Currently **readExtinction.getDistanceAtMag()** will just return the maximum distance for sight-lines in which the true maximum distance lies beyond the limit of the model.

I think this would work similarly to the following:

- Select healpix for which the returned distance is > the maximum distance at those healpix. For those hpids:
- Use the maximum E(B-V) in the sight-line (i.e. the [-1]'th E(B-V);
- Compute the distance directly from the max E(B-V) and the delta-magnitude requested.

I think all the above can be vectorized.

**Comment by willclarkson** [comment]
I have implemented this in method **readExtinctions.getDistanceAtMag()**, which now also returns the boolean array indicating whether the sight-line lay beyond the maximum in the model. All calls to **getDistanceAtMag()** have been updated in **readExtinctions.py** - I am testing the example notebook on Datalab now. 

It might be worth splitting off the relevant lines in **getDistanceAtMag()** into a new method, it depends on how widely this capability is likely to be used.
